src/wallet/api.rs

```diff
@@ -1327,15 +1366,17 @@ impl Wallet {
 
         // Total default: (16 + 2 + 4 + 4 + 1 + 16 = 43 WU + variable) WU
         // Source - https://docs.rs/bitcoin/latest/src/bitcoin/blockdata/transaction.rs.html#599-602
-        let base_weight = 43 + MAX_SPLITS as u64 * (target_weight.to_wu() + change_weight.to_wu());
+        let tx_base_weight = 43
```

Comment:
Can we turn this 43 into a constant? Put it at the start of the function. And keep the doc comment just above it. Like all other constants. The number is used in multiple spots, we can use the constant everywhere. 

Will be cleaner for future devs to understand where it comes from.